manifests/base/deployments/argocd-dex-server_patch.yaml

```diff
@@ -2,14 +2,20 @@ apiVersion: apps/v1
 kind: Deployment
 metadata:
   name: argocd-dex-server
+  annotations:
+    alpha.image.policy.openshift.io/resolve-names: '*'
 spec:
   template:
     spec:
       containers:
       - name: dex
+        image: argocd-dex-server
         env:
         - name: DEX_OAUTH_CLIENT_SECRET
           valueFrom:
             secretKeyRef:
               name: argocd-dex-server-oauth-token
               key: token
+      initContainers:
```

Comment:
What is this doing?

Reply:
it's insuring that the initcontainer from the upstream spec is forced to use the image from the openshift local registry, basically that's the entire pr, just re-directing images to use the imagestreams which pull into local registry instead of from docker